### PR TITLE
Closes #616 | Add Dataloader TyDIQA-ID-NLI

### LINKED ISSUE #616: Create dataset loader for TyDiQA-ID-NLI
Dataloader name: `tydiqa_id_nli/tydiqa_id_nli.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?tydiqa_id_nli

|     Dataset| tydiqa_id_nli |
|-------------|---|
| Description | The TyDIQA-ID-NLI dataset is derived from the TyDIQA-ID question answering dataset, utilizing named entity recognition (NER), chunking tags, regex, and embedding similarity techniques to determine its contradiction sets. Collected through this process, the dataset comprises various columns beyond premise, hypothesis, and label, including properties aligned with NER and chunking tags. This dataset is designed to facilitate Natural Language Inference (NLI) tasks and contains information extracted from diverse sources to provide comprehensive coverage. Each data instance encapsulates premise, hypothesis, label, and additional properties pertinent to NLI evaluation. |
| Subsets | Indonesian |
| Languages | ind |
| Tasks | Natural Language Inference |
| License | Apache license 2.0 (apache-2.0) |
| Homepage | https://huggingface.co/datasets/muhammadravi251001/tydiqaid-nli |
| HF URL | https://huggingface.co/datasets/muhammadravi251001/tydiqaid-nli |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by muhammadravi251001**
**Title**: Add Dataloader TyDIQA-ID-NLI

**First line PR Message**: Closes https://github.com/SEACrowd/seacrowd-datahub/issues/616

### Notes
- For the `_CITATION` field, I will add it later.

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/{my_dataset}/{my_dataset}.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `{my_dataset}` folder.
- [x] Provide values for the `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/tydiqa_id_nli/tydiqa_id_nli.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  doi       = {},
+  biburl    = {},
+  bibsource = {}
+}
+"""
+
+_DATASETNAME = "tydiqa_id_nli"
+
+# TODO: Add description of the dataset here
+# You can copy an official description
```
Comment (MJonibek) [inline seacrowd/sea_datasets/tydiqa_id_nli/tydiqa_id_nli.py:61]:
```suggestion

```

**Review by MJonibek — APPROVED** [review]
@muhammadravi251001 Checked, LGTM! Thank you for your work. Just small problem: can you delete unnecessary comment

**Comment by muhammadravi251001** [comment]
> @muhammadravi251001 Checked, LGTM! Thank you for your work. Just small problem: can you delete unnecessary comment 

Thanks for the review, Sir!

**Review by luckysusanto — CHANGES_REQUESTED** [review]
The citation has not been added yet to the code.
Otherwise, this looks good!

**Comment by muhammadravi251001** [comment]
> The citation has not been added yet to the code. Otherwise, this looks good!

Yup, the citation is still not provided because I still waiting for the workshop's notification.

Thanks for the review, Lucky!

**Comment by holylovenia** [comment]
Hi @luckysusanto, is there anything else that @muhammadravi251001 needs to address besides the pending `_CITATION`? I'd like to merge the dataloader if you don't find any other issues.

**Comment by luckysusanto** [comment]
Sorry for the slow response @holylovenia 
No, I don't have more to add. I was thinking of waiting until the citation is put before accepting.
If it is not an issue, I'll approve now >.<.
Thanks!

**Review by luckysusanto — APPROVED** [review]

**Pull request merged**